wp-content/civi-extensions/goonjcustom/goonjcustom.php

```diff
@@ -126,6 +129,35 @@ function goonjcustom_evaluate_tokens(TokenValueEvent $e) {
       }
 
       $row->tokens('contact', 'inductionDetails', $inductionDetailsMarkup);
+			// Fetch induction activity details
+			$inductionActivity = \Civi\Api4\Activity::get(FALSE)
+					->addSelect('activity_date_time', 'Induction_Fields.Goonj_Office')
+					->addWhere('activity_type_id:name', '=', 'Induction')
+					->addWhere('source_contact_id', '=', $contactId)
+					->setLimit(1)
```

Comment:
Why are we using ->setLimit(1) here when we already have ->first()? Do we really need it, or should it be removed?